e2e/test/scenarios/filters-reproductions/filters-reproductions.cy.spec.js

```diff
@@ -1400,3 +1400,37 @@ describe("issue 45877", () => {
     });
   });
 });
+
+describe("issue 47887", () => {
+  beforeEach(() => {
+    restore("setup");
+    cy.signInAsAdmin();
+  });
+
+  it("Case expression with type/Date default value and type/DateTime case value has Date filter popover enabled (metabase#47887)", () => {
+    visitQuestionAdhoc({
```

Comment:
I'm not sure whether this is the right approach here. Would welcome FE eyes on this.